assets/js/components/ClusterDetails/ChecksResultOverviewNew.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { format } from 'date-fns';
+
+import CheckResultCount from './CheckResultCount';
+import Spinner from '@components/Spinner';
+
+
+const ChecksResultOverviewNew = ({
+  data,
+  error = null,
+  loading = false,
+  onCheckClick,
+}) => {
+  if (loading || data.status === 'running') {
```

Comment:
Super nice!
Maybe we could involve @jagabomb here once this is merged to think about UX options for loading/error/not checks available scenarios (not discarding the current implication)

Reply:
I agree, i did some impro but it's not the best UI :D